### PR TITLE
TASK: Update dev dependencies

### PR DESCRIPTION
**Pull request by kdambekalns**
Those dev dependencies were a tad bit outdated…

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,17 +23,17 @@
...
     "require-dev": {
-        "neos/buildessentials": "~4.1.0",
-        "mikey179/vfsstream": "~1.6",
-        "phpunit/phpunit": "~6.0.0",
+        "neos/buildessentials": "^5.0",
+        "mikey179/vfsstream": "^1.6",
+        "phpunit/phpunit": "^7.0",
         "symfony/css-selector": "~2.0",
-        "neos/behat": "dev-master"
+        "neos/behat": "^4.1"
```
Comment (aertmann) [inline composer.json]: 5.0 no?

Comment (kitsunet) [reply composer.json]: good catch

**Comment by kdambekalns** [comment]
Maybe @kitsunet can have another quick look?

**Review by kitsunet — APPROVED** [review]
Looks good

**Pull request merged**